utils/java/mvn.go

```diff
@@ -42,37 +43,39 @@ var mavenDepTreeJar []byte
 
 type MavenDepTreeManager struct {
 	DepTreeManager
-	isInstalled     bool
-	cmdName         MavenDepTreeCmd
-	settingsXmlPath string
+	isInstalled         bool
+	isCurationCmd       bool
+	curationCacheFolder string
```

Comment:
```suggestion
	isCurationCmd       bool
	curationCacheFolder string
```
Please add a comment above to describe why and what are each attribute